Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
I mean, you can talk to @zkat about it if you like. Kat cut the most recent two `npm` releases and is around the thread, so.

`make install` essentially just throws the job back to `npm` though and says _"install yourself with yourself"_ so it makes perfectly natural sense that `npm` behaves this way. Without diving into whether or not I agree with the behaviour I'm pretty much 100% certain it's behaving as-expected and as intended upstream.